backend/sass/common.blocks/_setup.scss

```diff
@@ -1,23 +1,45 @@
 // The wallet ui:
 @import '../include/defs';
 
-.wallet {
-}
-
 $fullscreen-wallet-grey: #f1f1f1;
 $wallet-top-btm-margin: 0.5rem 0 0.5rem 0;
 $kadena-orange: #F3AB3C;
 
-.fullscreen {
+.setup__fullscreen {
   display: grid;
   grid-template-columns: 1fr 1fr 1fr 1fr;
   grid-template-rows: 0.2fr 1.6fr;
   grid-template-areas: "back header header ." ". wrapper wrapper .";
+
+  p {
+    margin: 2rem auto;
+  }
+
+  .checkbox {
```

Comment:
That's not how BEM is supposed to work. Instead of overriding another block within this block, you would create an element `setup__checkbox`, that is, if you need customizations that are really only relevant in the `setup` block. 

Alternatively, if this is a more general purpose variant of a `checkbox`, you could move that code to the `checkbox` file by means of a modifier class like `checkbox_align_left` or something.

You would also not nest things but have a top level definition:

```
.setup__checkbox {
...
```
in this file or

```
.checkbox_align_left {
....
```

in _checkbox.scss. In HTML you then simply apply all the needed classes, e.g.:

```
elClass "div" "checkbox checkbox_align_left" $ ....
```

By adhering to BEM standards we should actually have maintainable CSS, even in the long run. That's why I moved over to it, from the mess we had before.

A good resource on BEM can be found [here](https://en.bem.info/methodology/quick-start/).